Monika After Story/game/script-greetings.rpy

```diff
@@ -371,3 +371,55 @@ label monikaroom_greeting_post:
         set_keymaps()
     return
 
+label greeting_youarereal:
+default persistent.you = True
+python:
+    try:
+        renpy.file("/characters/" + persistent.playername + ".chr")
+            persistent.you = True
+        except IOError:
```

Comment:
Remove `IOError` from `except`. We can just assume that if any error occurs when attempting to find the file, it doesnt exist. No reason to catch a specific set of exceptions.